src/main/java/fr/xephi/authme/process/email/AsyncAddEmail.java

```diff
@@ -57,6 +62,13 @@ public void addEmail(Player player, String email) {
             } else if (!validationService.isEmailFreeForRegistration(email, player)) {
                 service.send(player, MessageKey.EMAIL_ALREADY_USED_ERROR);
             } else {
+                EmailChangedEvent event = bukkitService.createAndCallEvent(isAsync
+                    -> new EmailChangedEvent(player, null, email, isAsync));
+                if (event.isCancelled()) {
+                    ConsoleLogger.warning("Could not add email to player '" + player + "' – event was cancelled");
```

Comment:
Like @ljacqu said, warning is reserved for unexpected situation that should be investigated. Please change it to something else like info.